migrating_from_ocp_3_to_4/premigration-checklists.adoc

```diff
@@ -0,0 +1,90 @@
+* [ ] You have checked for pending certificate requests by running the following command:
++
+[source,terminal]
+----
+$ oc get csr -A | grep pending -i
+----
+
+* [ ] The link:https://docs.openshift.com/container-platform/3.11/install_config/configuring_authentication.html#overview[identity provider] is working.
+
+[id="target-cluster-checklist_{context}"]
+== Target cluster checklist
+
+* [ ] The {mtc-short} version is the same on all clusters.
+* [ ] All xref:../migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc#migration-prerequisites_migrating-applications-3-4[{mtc-short} prerequisites] are met.
+* The cluster meets the minimum hardware requirements for the specific platform and installation method, for example, on xref:../installing/installing_bare_metal/installing-bare-metal.adoc#minimum-resource-requirements_installing-bare-metal[bare metal].
+* [ ] The cluster has xref:../storage/dynamic-provisioning.adoc#defining-storage-classes_dynamic-provisioning[storage classes] defined for the storage types used by the source cluster, for example, block volume, file system, or object storage.
++
+[NOTE]
+====
+NFS does not require a defined storage class.
+====
+
+* [ ] The cluster has the correct network configuration and permissions to access external services, for example, databases, source code repositories, container image registries, and CI/CD tools.
+* [ ] External applications, services, and appliances that use services provided by the cluster have the correct network configuration and permissions to access the cluster.
+* [ ] Internal container image dependencies are met.
++
+If an application uses an internal image in the `openshift` namespace that is not supported by {product-title} {product-version}, you can manually update the xref:../migrating_from_ocp_3_to_4/migrating-applications-3-4.adoc#migration-updating-deprecated-internal-images_migrating-applications-3-4[{product-title} 3 image stream tag] with `podman`.
+* [ ] The target cluster and the replication repository have sufficient storage space
+* [ ] The xref:../authentication/understanding-identity-provider.adoc#supported-identity-providers[identity provider] is working.
+
+[id="performance-checklist_{context}"]
+== Performance checklist
+
+* [ ] The migration network has a minimum throughput of 10 Gbps.
+* [ ] The clusters have sufficient resources for migration.
++
+[NOTE]
+====
+Clusters require additional memory, CPUs, and storage in order to run a migration on top of normal workloads. Actual resource requirements depend on the number of Kubernetes resources being migrated in a single migration plan. You must test migrations in a non-production environment in order to estimate the resource requirements.
+====
+* [ ] The xref:../support/troubleshooting/verifying-node-health.adoc#reviewing-node-status-use-and-configuration_verifying-node-health[memory and CPU usage] of the nodes is healthy.
```

Comment:
Should "is" be "are" since there are two usage items? Not sure if the wording is considering it to be one thing or not here.